src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -90,12 +104,14 @@ class ThemeColorsEditor extends React.Component {
                 >
                   <span
                     className="theme-unit__swatch"
-                    style={{ backgroundColor: colorToCSS(color) }}
-                    title={colorLabels[name]}
+                    style={{backgroundColor: color ? colorToCSS(color) : "white", borderStyle: color ? "solid" : "dashed"}}
```

Comment:
`"solid"` -> `""` to fall back to the default stylesheet.

But why not this instead?

```
color ? {backgroundColor: ...} : {borderStyle: "dashed"}
```

(code golfing isn't necessary; if readability improves by using multiple lines or even a separate variable, by all means use it.)